bundle/edu.gemini.model.p1/src/main/scala/edu/gemini/model/p1/overheads/Overheads.scala

```diff
@@ -0,0 +1,130 @@
+package edu.gemini.model.p1.overheads
+
+import java.time.Duration
```

Comment:
Not sure if I should be using `Duration` here due to the weirdness of `Duration.ofMinutes(59).toHours.getValue()` returning 0, but it seemed like a good idea before I figured that out.

Leaving it as is for now.